cmake/PackageConfig.cmake.in

```diff
@@ -0,0 +1,38 @@
+@PACKAGE_INIT@
+
+#@PROJECT_NAME@-config.cmake
+#
+# Imported interface targets provided:
+#  * @PROJECT_NAME@::@PROJECT_NAME@ - library target
+
+# Include targets file.  This will create IMPORTED target @PROJECT_NAME@
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
+include(CMakeFindDependencyMacro)
+
+find_dependency(MPI)
+
+# ON/OFF implies @PROJECT_NAME@ was compiled with/without OpenMP
+if(@OPENMP@)
+  find_dependency(OpenMP COMPONENTS Fortran)
+endif()
+
+find_dependency(NetCDF COMPONENTS Fortran)
+
+find_dependency(sigio CONFIG)
+find_dependency(w3nco CONFIG)
+find_dependency(g2 CONFIG)
+find_dependency(g2tmpl CONFIG)
+find_dependency(bacio CONFIG)
+find_dependency(ip CONFIG)
+find_dependency(sp CONFIG)
+find_dependency(sfcio CONFIG)
+find_dependency(nemsio CONFIG)
+find_dependency(gfsio CONFIG)
+find_dependency(w3emc CONFIG)
+find_dependency(crtm CONFIG)
```

Comment:
``nceppost::nceppost`` library target does not depend on ``gfsio``, ``nemsio``, ``sfcio``, and ``sigio``.

Reply:
resolved in 92e1d4d